web/blog/tenzir-v4.5/index.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Tenzir v4.5
+authors: [dominiklohmann]
+date: 2023-11-17
+tags: [release, operators, expression, index, api, demo-node]
+comments: true
+---
+
+Here comes [Tenzir v4.5](https://github.com/tenzir/tenzir/releases/tag/v4.5.0)!
+This release ships a potpourri of smaller improvements that result in faster
+historical query execution and better deployability.
+
+![Tenzir v4.5](tenzir-v4.5.excalidraw.svg)
+
+<!-- truncate -->
+
+## More Robust Numeric Query Expressions
+
+Ever scratched your head what numeric literal to use in a query so that you hit
+the right fields in events? No more! You can now write `x < 42`, `x < +42`, or
+`x < 42.0` in an expression to filter events where the field `x` is less than
+42, regardless of whether `x` is of type `uint64`, `int64`, or `double`.
+Previously, you had to know the exact number type for the expression to bind
+properly to the event schema.
+
+This incremental improvement is part of a larger thrust to improve the language.
+We have plans to convert additional literals during expression bindings to
+provide deeper reach into the data, without users really having to deal with
+schemas.
+
+## Sparse Indexes for Number Types
+
+Tenzir's storage engine builds sparse indexes per partition. These sketch
+data structures, like Bloom filters and min-max synopses, accelerate historical
+queries by ensuring that only relevant partitions get loaded into memory.
+
+We've now added additional sketches for types `bool`, `time`, `duration`,
+`int64`, `uint64`, and `double` to accelerate a broader range of queries. For
+eaxmple, this query now runs faster:
+
+```
+export
+| where :timestamp > 1 hour ago && dest_port == 80
+```
+
+## The API as Operator
+
+We exposed the API as a new operator called
+[`api`](/next/operators/sources/api). The benefit primarily materializes for
+developers who can now rapidly prototype integrations by using the app or
+`tenzir` command line tool, without having to do spin up the integrated web
+server and do gymnastics with `curl` and `jq`.
+
+TODO: provide an example
```

Comment:
@dominiklohmann do you have a cool example?

Reply:
I added two.